README.md

```diff
@@ -1 +1,7 @@
-# charts
+# Treeverse Helm Charts Registry
+
+This repository serves as a collection of Helm Charts, including those used by various projects under Treeverse.
+
+- [lakeFS](https://github.com/treeverse/charts/tree/master/charts/lakefs)
+
+When preparing for a release, please update the CHANGELOG.md file with the detailed list of changes made since the last version.
```

Comment:
```suggestion
When preparing for a release, please update the CHANGELOG.md file with the detailed list of changes made since the last version.

After the chart is released, please update the release notes with the changes from the CHANGELOG.md file.
```